bootstrap4/.js/src/main/scala/io/udash/bootstrap/datepicker/UdashDatePicker.scala

```diff
@@ -37,6 +37,38 @@ final class UdashDatePicker private[datepicker](
   ).render
   private val jQInput = jQ(inp).asInstanceOf[UdashDatePickerJQuery]
 
+  locally {
+    registerSetupCallback(componentId, () => {
+      jQInput.datetimepicker(
+        optionsToJsDict(options.get)
+          .setup(optionsDict => date.get.foreach(date => optionsDict.update("date", dateToMoment(date))))
+      )
+
+      nestedInterceptor(new JQueryOnBinding(jQInput, "change.datetimepicker", (_: Element, event: JQueryEvent) => {
+        val dateOption = event.asInstanceOf[DatePickerChangeJQEvent].option
+          .flatMap(ev => sanitizeDate(ev.date))
+          .map(momentToDate)
+        val oldDateOption = date.get
```

Comment:
Is this safe?

Reply:
I believe it is - this `get` is invoked on a `RedableProperty`, not an `Option`.